portal/src/texts/profile/picture.md

```diff
@@ -1,5 +1,288 @@
+
+1. **bilde: naturlig, uttryksfullt ansikt**
+
+Personene skal ikke se ut som profesjonelle modeller. Unngå klisjeaktige, unaturlige poseringer og påtvungne uttrykk.
+
+2. [bilde: modell/klisjé]
+
+La gjerne personene engasjere seg i omgivelsene, uoppmerksom på fotografen. De kan for eksempel utføre dagligdagse gjøremål som å handle, gå tur, kommunisere med andre, observere noe eller delta i en aktivitet.
+
+Med to eller flere personer i bildet kan det være enklere å illustrere mellommenneskelige relasjoner og skape dynamikk i bildet.
+
+3. [bilde: to personer “fanget i øyeblikket”]
+
+Fremtind har kunder på tvers av alder, kjønn, etnisitet, religion, legning, helse og kroppstype. Alle disse menneskene skal kunne relatere seg til situasjoner i bildene.
+
+[bilde: Gode eksempler på noen av disse uten å stereotype]
+
+Bilder er en god flate å bruke til å utfordre stereotyper og vise at vi er et fremtidsrettet selskap.
+
+
+[bilde: far/barn, kvinne/annleggsarbeider. Bedre forslag???]
+
+Hvis vi tar dronebilder med mennesker er det viktig at situasjonen er tydelig og oversiktlig.
+
+[bilde: drone ryddig vs kaotisk]
+
+- Ekte mennesker
+-	Hverdagslige situasjoner
+-	Flere personer skaper dynamikk
+	•	Vise mangfold
+	•	Utfordre stereotyper
+	•	Oversiktlige situasjoner i dronefoto
+
+## Omgivelser
+
+Folk skal kjenne seg igjen i bildene. Vi  bruker bilder med nordiske omgivelser eller som ikke viser stedstilhørighet, men som føles kjent. Vi behøver ikke, men det er ok om vi kan gjenkjenne om vi er i Haugesund, Alta eller Fetsund.
+
+[bilde: gjenkjennelig sted + ikke gjenkjennelig, men nordisk]
+
+Selvom omgivelsene er nordiske skal menneskene representere ulike folketyper og kulturer og vise mangfoldet i det norske samfunnet.
+
+[bilde: hjemme hos utenlandsk famile, med utsikt over oslo og melk på bordet el.]
+
+Vi ønsker å variere mellom omgivlelser som viser by, land og kyst i tillegg å veksle mellom bilder i inne- og utemiljø.
+
+Vi tar bilder i all slags vær, og skifter gjerne bilder etter årstidene på de flatene det passer.
+Høytider - norske og andre.
+
+Vi unngår bilder med elementer som avslører at bildene er tatt i et annet land. For eksempel utsikt med unorsk landkap eller bysilhuette, merkevarer som er uvanlige i Norge eller innslag av tekst eller skilt på andre språk.
+
+[bilde: riktig + med feil skilting og dashboard]
+
+	•	Gjenkjennelige omgivelser
+	•	Vi viser mangfoldet i det norske samfunnet
+	•	Varierte omgivelser
+	•	Vi skifter gjerne bilder med årstidene og høytider
+	•	Se opp for fremmedgjørende elementer
+
+## Kategoribilder
+
+Når vi snakker om et konkret forsikringsprodukt kan det være vanskelig å finne bilder som passer perfekt. Her er noen tips og retningslinjer til de ulike forsikringene. Felles for kategoribildene er at vi ønsker fokus på en situasjon som oppstår rundt eller i samspill med produktet fremfor produktet i seg selv.
```

Comment:
Bruk heller  "tingen som er forsikret" i stedet for produkt. Du bruker forsikringsprodukt tidligere i teksten og det kan være forvirrende...
(Line)

Reply:
Jeg har ikke rettet dette da jeg språkvasket teksten, Ida, siden jeg så kommentaren nå etterpå, så da kan du vurdere om du vil endre det til "tingen som er forsikret".